product_security/red_hat_responsibilities/logging_metrics_monitoring.adoc

```diff
@@ -0,0 +1,75 @@
+[[dedicated-security-logging-metrics-monitoring]]
+= Logging, Metrics, and Monitoring
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+{product-title} uses an EFK stack to collect logs from applications and present
+them to OpenShift users. {product-title} administrators can view all application
+and project logs, but application developers can only view logs for projects
+they have permission to view.
+
+The EFK stack consists of the following components:
+
+* _Elasticsearch_: Object store where logs are stored and provides the ability to search logs.
+* _Fluentd_: Gathers and sends logs to Elasticsearch; this runs on all nodes on the cluster.
+* _Kibana_ : A web interface to allow for easy interaction with Elasticsearch.
+* _Curator_: Schedule Elasticsearch maintenance operations automatically.
+
+By default, {product-title} retains a maximum of 14 days or 200GB of application
+logs, whichever comes first. Increased logging rates and large log sizes can
+require shorter log retention before logs are rotated.
+
+Kibana can be accessed at:
+
+----
+https://logs.[cluster-name].openshift.com
+----
+
+[[security-audit-logging]]
+== Audit Logging
+
+Audit logs are maintained by Red Hat for Compliance purposes and are not
+shareable with customers at this time.
+
+[[security-metrics]]
+== Metrics
+
+Metrics provids the ability to view CPU, memory, and network-based metrics in
+the {product-title} web console. These metrics allow for the horizontal
+autoscaling of pods based on parameters provided by an {product-title} user.
+
+The metrics stack consists of the following components:
+
+* _Hawkular Metrics backed by Cassandra_: A metrics engine which stores the data
+persistently in a Cassandra database. When this is configured, CPU, memory and
+network-based metrics are viewable from the OpenShift Dedicated web console and
+are available for use by horizontal pod autoscalers.
+* _Heapster_: A service that retrieves a list of all nodes from the master server,
+then contacts each node individually and scrapes the metrics for CPU, memory,
+and network usage. It then exports these into Hawkular Metrics.
+
+[[customer-controlled-monitoring]]
```

Comment:
This is my bad. The Customer controlled monitoring needs to go under Recommendations for customers, with Logging.